xbmc/utils/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(SOURCES ActorProtocol.cpp
             AlarmClock.cpp
             AliasShortcutUtils.cpp
+            AMLUtils.cpp
```

Comment:
either github is playing tricks on me or AMLUtils.cpp/h is still outside this IF condition